assets/shells/config.bat

```diff
@@ -13,6 +13,10 @@ set {{$K}}={{$V}}
 DOSKEY {{$K}}=state run {{$CMD}}
 {{end}}
 
+{{ if .ExecAlias }}
+DOSKEY state="{{.ExecAlias}}" $*
```

Comment:
@Naatan My fix here makes me think that the doskey usage above for scripts is broken. If it looks so to you as well, I'll file a bug.

Reply:
This is probably unique to DOSKEY right? I think alias does this implicitly. Feel free to address in this PR since it's such a small change.

Reply:
I successfully tested this manually as compared to what's on master.